### PR TITLE
40.1 migration guide review

### PR DESCRIPTION
**Pull request by Witoso**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Update to 40.1 fixes.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/updating/update-to-40.md [hunk 1]
```diff
@@ -16,10 +16,16 @@ modified_at: 2023-11-07
...
 ## Update to CKEditor&nbsp;5 v40.1.0
 
-### New way to handle images inserted into content
+### Changes to the default insert image action
 
+We changed how the images are inserted by default. For long time, the image insert action detected where the selection is placed, and then inserted an inline image or a block one. This caused confusion in some cases, and led to suboptimal experience. From now on, the image will be inserted as a block one by default.
+
+This of course depends on how an integrator set up the editor. If only `ImageInline` plugin is used, the editor will automatically insert images as inlined. If you load the `Image` plugin, and wish to restore the previous behaviorm you only need to change the config.
+
+// Not sure if section below is needed:
```
Comment (Witoso) [inline docs/updating/update-to-40.md:25]: as above, maybe it should be a part of Image guide?

File: docs/updating/update-to-40.md [hunk 2]
```diff
@@ -16,10 +16,16 @@ modified_at: 2023-11-07
...
 Inline images can be inserted in the middle of a paragraph or a link just like regular text. Block images, on the other hand, can be inserted only between other blocks like paragraphs, tables, or media. Being larger and existing as standalone content, block images can also have individual captions. Other than that, both types of images can be resized, linked, etc.
 
+// This config was aleardy there, we just renamed `undefined` to `auto`.
```
Comment (Witoso) [inline docs/updating/update-to-40.md:28]: Requires rewrite, as the config already existed.

File: docs/updating/update-to-40.md [hunk 3]
```diff
@@ -16,10 +16,16 @@ modified_at: 2023-11-07
...
 
-### New way to handle images inserted into content
+### Changes to the default insert image action
 
+We changed how the images are inserted by default. For long time, the image insert action detected where the selection is placed, and then inserted an inline image or a block one. This caused confusion in some cases, and led to suboptimal experience. From now on, the image will be inserted as a block one by default.
+
+This of course depends on how an integrator set up the editor. If only `ImageInline` plugin is used, the editor will automatically insert images as inlined. If you load the `Image` plugin, and wish to restore the previous behaviorm you only need to change the config.
+
+// Not sure if section below is needed:
 Inline images can be inserted in the middle of a paragraph or a link just like regular text. Block images, on the other hand, can be inserted only between other blocks like paragraphs, tables, or media. Being larger and existing as standalone content, block images can also have individual captions. Other than that, both types of images can be resized, linked, etc.
```
Comment (Witoso) [inline docs/updating/update-to-40.md:26]: Not sure if this section is needed, as we didn't change anything here.

File: docs/updating/update-to-40.md [hunk 4]
```diff
@@ -16,10 +16,16 @@ modified_at: 2023-11-07
...
 
+// This config was aleardy there, we just renamed `undefined` to `auto`.
 Introduced in the latest version comes a handy `image.insert.type` configuration setting that lets the integrators set up the way newly uploaded ar pasted images are handled in the editor content. By default, if the `image.insert.type` configuration is not specified, all images inserted into the content will be treated as block images. This means that inserting an image inside a paragraph (or other content blocks) will create a new block for the image immediately below or above the current paragraph or block. After insertion, you can transform the block image into an inline image using the {@link features/images-overview#image-contextual-toolbar contextual toolbar}.
```
Comment (Witoso) [inline docs/updating/update-to-40.md:29]: Needs, rewrite, config was there.

**Review by godai78 — APPROVED** [review]
LGTM

File: docs/updating/update-to-40.md [hunk 5]
```diff
@@ -16,10 +16,16 @@ modified_at: 2023-11-07
 
 ## Update to CKEditor&nbsp;5 v40.1.0
 
-### New way to handle images inserted into content
+### Changes to the default insert image action
 
+We changed how the images are inserted by default. For long time, the image insert action detected where the selection is placed, and then inserted an inline image or a block one. This caused confusion in some cases, and led to suboptimal experience. From now on, the image will be inserted as a block one by default.
+
+This of course depends on how an integrator set up the editor. If only `ImageInline` plugin is used, the editor will automatically insert images as inlined. If you load the `Image` plugin, and wish to restore the previous behaviorm you only need to change the config.
+
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/updating/update-to-40.md:24]: Trimmed down this wall of text as the bottom info is repeated under the snippet anyway.

**Pull request merged**